README.md

```diff
@@ -3,8 +3,54 @@
 This a pure Scala implementation of [Babelfish](https://doc.bblf.sh/) client.
 It uses [ScalaPB](https://scalapb.github.io/grpc.html) for Protobuf/gRPC code generation.
 
+### Installation
+
+```
+./sbt assembly
+```
+
+gRPC/protobuf are re-generate from `src/main/proto` on every `./sbt compile` and
+are stored under `./target/scala-2.11/src_managed/`. 
```

Comment:
After I added `crossPaths := false` to the build the `scala-2.11` part doesn't exist anymore, so it'd be `./target/srcd_manager/`.